content/5_Plat/Centroid.mdx

```diff
@@ -17,13 +17,85 @@ TODO:
 
 ## Introduction
 
+### Centroids
+
+A **centroid** of a tree is defined as a node such that when the tree is rooted at it, no other nodes
+have a subtree of size greater than $\frac{N}{2}$.
+
 <FocusProblem problem="sam" />
 
-**Centroid Decomposition** is a divide and conquer technique for trees. The
-**centroid** of a tree is a node which, if rooted, results in no other nodes
-having a subtree of size greater than $\frac{N}{2}$. **Centroid Decomposition**
-works by repeated splitting the tree and each of the resulting subgraphs at the
-centroid, producing $\mathcal{O}(\log N)$ layers of subgraphs.
+<LanguageSection>
+
+<CPPSection>
+
+We can find a centroid in a tree by starting at the root. Each step, loop
+through all of its children. If all of its children have subtree size less
+than or equal to $\frac{N}{2}$, then it is a centroid. Otherwise, move to the
+child whose subtree is more than $\frac{N}{2}$ and repeat until you find a
```

Comment:
> whose subtree is more than $\frac{N}{2}$

subtree size is greater than $\frac{N}{2}$?

also pls re-request review next time